include/taco/format.h

```diff
@@ -65,10 +93,35 @@ class Format {
   /// Sets the types of the coordinate arrays for each level
   void setLevelArrayTypes(std::vector<std::vector<Datatype>> levelArrayTypes);
 
+  /// Return true if Format represents a blocked tensor format
+  bool isBlocked();
+
+  /// Gets the number of nested block levels in the format
+  int numberOfBlocks();
```

Comment:
Maybe rename to something like `numBlockLevels`?  The fact that this is referring to the number of levels doesn't really get across in the current name.